buildconfig/CMake/PythonPackageTargetFunctions.cmake

```diff
@@ -65,15 +69,22 @@ CustomInstallLib = patch_setuptools_command('install_lib')
     # --install-scripts=bin --install-lib=lib removes any of the platform/distribution specific install directories so we can have a flat
     # structure
     install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${_setup_py} install -O1 --single-version-externally-managed --root=${_setup_py_build_root}/install --install-scripts=bin --install-lib=lib WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})")
-    # register the "installed" components with cmake so it will carry them over
-    install(DIRECTORY ${_setup_py_build_root}/install/lib
-            DESTINATION .
+
+    # Registers the "installed" components with CMake so it will carry them over
```

Comment:
I think this will break the Linux packaging. For Linux we want to carry over the `lib` directory as it is into the root of the installation but for Windows (and ultimately Mac) we need it in the `bin` directory for now. I think the best way to handle this for now would be an `if(WIN32)` do the behaviour you have here and `else` do the old behaviour.